index.html

```diff
@@ -15,29 +17,31 @@
     <i class='bx bx-menu bx-sm'></i>
   </nav>
   <section class="headline">
+    <div class="nav">
+      <ul><li>Portoflio</li>
+        <li>About</li>
+        <li>contact</li>
+      </ul>
+    </div>
```

Comment:
Great job on addressing the previous reviewer's requests and making improvements to your project. However, there is still an issue with the navigation items not displaying properly in different screen sizes, especially in the 1044px screen size.

To ensure a consistent and responsive user experience, please review your code and make the necessary adjustments to ensure that the navigation items are properly displayed and accessible at all screen sizes. Consider using responsive techniques such as media queries and adjusting the layout to accommodate different screen sizes.

By addressing this issue and ensuring the proper visibility and functionality of the navigation items across various screen sizes, you will provide a seamless navigation experience for your users. Keep up the good work! 👍

![image](https://github.com/tsigewossne/portfolio/assets/55840999/2ec9daa5-f6e0-4f6f-b877-c792d36d62d2)